### PR TITLE
Polished the example action

### PR DESCRIPTION
**Pull request by GMaiolo**
- Running only on master branch
- Renamed `test` to `deploy-storybook`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,12 @@ In your git repository, create a file `.github/workflows/chromatic.yml` with the
...
+on:
+  push:
+    branches: [ master ]
 
 jobs:
-  test:
+  deploy-storybook:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
```
Comment (jonniebigodes) [inline README.md:21]: @GMaiolo we have a little "breathing room" (pardon the bad pun) and we could probably go with:
```yml
jobs:
    chromatic-deployment:
         runs-on: ubuntu-latest
         steps:
            - uses:actions/checkout@v1
```
Reasoning behind this is that publishing Storybook one part of the task. We're also doing Chromatic's automation process and setting visual regression testing.

Let me know if you're ok with

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@GMaiolo Thank you very much for the time and effort you put into this.

Left one note regarding the pull request let me know if you're ok with it and we'll move from there.

@ndelangen let me know as well if you're ok with it.

Stay safe

File: README.md (hunk 1 above)
Comment (GMaiolo) [reply README.md]: No worries @jonniebigodes
I just updated it

**Comment by GMaiolo** [comment]
@jonniebigodes
Just updated the name 👍

File: README.md (hunk 1 above)
Comment (jonniebigodes) [reply README.md]: @GMaiolo fantastic. Once again thank your making the changes so promptly. From my end i'm good. If @ndelangen signs up on it we'll merge it.

Stay safe

**Comment by GMaiolo** [comment]
There goes my contribution I guess

**Comment by ndelangen** [comment]
@ghengeveld can we port this the new repo?

**Pull request closed without merging**